tox.ini

```diff
@@ -6,11 +6,10 @@ envlist =
 [testenv]
 deps = flake8==3.6.0
 commands =
-    python setup.py test -q
+    python -m unittest discover pyflakes
+    flake8 --version
```

Comment:
Is this left over from debugging?

Reply:
seems it was an implicit conflict with https://github.com/PyCQA/pyflakes/pull/398/files#diff-b91f3d5bd63fcd17221b267e851608e8L10 from #398